docs/components/motor/gpio/encoded-motor.md

```diff
@@ -129,13 +126,15 @@ In addition to the [attributes for a non-encoded motor](/components/motor/gpio),
 | Name | Type | Inclusion | Description |
 | ---- | ---- | --------- | ----------- |
 | `encoder` | string | **Required** | `name` of the encoder. |
-| `ticks_per_rotation` | string | **Required** | Number of ticks in a full rotation of the encoder and motor shaft. |
-| `ramp_rate` | number | Optional | How fast to ramp power to motor when using RPM control. 0.01 ramps very slowly; 1 ramps instantaneously. Range is (0.0, 1.0]. <br> Default = `0.2` |
+| `ticks_per_rotation` | int | **Required** | Number of ticks in a full rotation of the encoder and motor shaft. |
+| `ramp_rate` | float | Optional | Rate to increase the motor's input voltage (power supply) per second when increasing the speed the motor rotates (RPM). <br> Range = `(0.0, 1.0]` <br> Default = `0.2` |

```

Comment:
I think the very slowly versus instantaneously part is very helpful and should stay in. Also not sure that the range should be in backticks since if you put ”(0.0, 1.0]” literally into your config it wouldn’t work.

Reply:
Hmm they could think 1 means take maximum time to ramp and 0 means take 0 time to ramp. idk